mlrun/feature_store/retrieval/spark_merger.py

```diff
@@ -252,7 +253,7 @@ def _get_engine_df(
                 )
             source_kind = target.kind
             source_path = target.get_target_path()
-
+            source_kwargs = target.source_attributes
```

Comment:
Not sure what to suggest here, but it's a bit confusing that something generic-sounding like `source_attributes` is only used in Spark.

Reply:
rename to `source_spark_attributes`